### PR TITLE
Project Execution - O1

### PR DESCRIPTION
**Pull request by McDoxa**

### CODE REVIEW

File: project-javascript-o1-promises/Oluwaseyi.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Washing Dishes</title>
+</head>
+<body>
+    <script>
+        /* Create a function and implement its logic after 1sec. */
+        const numDishes = Math.floor(Math.random()*10);
+        const wash = (resolve, reject) => {
+            if(numDishes <= 7 && numDishes > 0){
```
Comment (basir) [inline project-javascript-o1-promises/Oluwaseyi.html]: wrap it in setTimeout

File: project-javascript-o1-promises/Oluwaseyi.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                    reject('Wash: Fail');
+                }
+            }
+        //setTimeout(wash, 1000);
+        /* Create a new Promise with 'wash' parameter */
+        const newPromise = new Promise(wash);
+        /* Use .then() to get the number of dishes. */
+        newPromise.then( resolveValue => {
+            console.log('Number of Dishes: ' + resolveValue);
+            const dry = () => {
```
Comment (basir) [inline project-javascript-o1-promises/Oluwaseyi.html]: pass numDishes as a parameter to dry function

File: project-javascript-o1-promises/Oluwaseyi.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                                console.log('Dry: Fail');
+                                reject('too many dishes in the dryer')
+                                }
+                             }, 2000);
+                         })
+                     }
+            dry().then(numDishes => {
+                console.log("Done " + numDishes + " Dishes.");
+                    },returnedValue => {console.log('Sorry, ' + returnedValue)});
+            }, rejectValue => {console.log(rejectValue)}
```
Comment (basir) [inline project-javascript-o1-promises/Oluwaseyi.html]: use .catch not like this

**Comment by McDoxa** [comment]
Reviewed changes have been effected . Thank you

**Pull request merged**